src/components/Differential/DifferentialSearch.jsx

```diff
@@ -292,6 +296,54 @@ class DifferentialSearch extends Component {
     }
   };
 
+  getResultsColumnTooltips = (study) => {
+    const { onSetDifferentialResultsColumnTooltips } = this.props;
+    cancelGetDifferentialResultsColumnTooltips();
+    let cancelToken = new CancelToken((e) => {
+      cancelGetDifferentialResultsColumnTooltips = e;
+    });
+    omicNavigatorService
+      .getResultsColumnTooltips(study)
```

Comment:
unfortunately getting the column header tooltips requires a call (passing study)